redditAPI.py

```diff
@@ -3,6 +3,8 @@
 import os
 from dotenv import load_dotenv
 import time
+import re
```

Comment:
Please delete this import if not using this library.

Reply:
Forgot about it, Thanks will do and recommit!